node-core/services/registry/registry.go

```diff
@@ -151,3 +156,13 @@ func (s *Registry) FetchService(service interface{}) error {
 	}
 	return errUnknownService
 }
+
+func (s *Registry) CommitMultiStore() store.CommitMultiStore {
+	var cometService *cometbft.Service
```

Comment:
```suggestion
	var cometService cometbft.Service
```
try initializing cometService as the empty struct

Reply:
Not sure about empty struct but has to be double pointer iirc for fetch

Reply:
not sure its needed, works without issue for me